openmsupply/application-manager-web-app
# #800 Verification UI

Fixes #800

Requires [back-end PR #424](https://github.com/openmsupply/application-manager-server/pull/424)

Creates route (`/verify`) and basic UI for verification page. Note that this route is outside the "Authenticated Content" wrapper, as it needs to be accessible to non-logged in users.

To test:
- follow instructions from the back-end PR
- notice that if you try and visit a verification page more than once you will get an error UI instead.

FYI -- the layout is basically just a copy of the "Submission" page. Keeping it simple.
